linode/manager
# Feat: [M3-5540] Table rows hover state + Table v7 stories

## Description 📝
This ticket implements a new interactive style for table rows and migrates mdx stories to v7 stories for the Table components

## Major Changes 🔄
**List highlighting major changes**
- Adds an hover state on table rows for all tables (it was only the Linodes table in the past)
- Moves base components styles to Styled Components
- Refactor MDX stories to v7 stories for `Table`, `TableRowLoading`, `TableRowEmpty`, `TableRowError`

## Preview 📷
**Include a screenshot or screen recording of the change**

> **Note**: Use `<video src="" />` tag when including recordings in table

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/130582365/acdcc18a-4725-4b63-8d27-c2b6f844df08" /> | <video src="https://github.com/linode/manager/assets/130582365/2a794ef6-8179-4320-af07-9e1ace7b8fa4" /> |

## How to test 🧪
1. Pull code locally and verify the new table row hover style
2. Verify new stories in storybook
3. Confirm no regression on the table components
